src/loader.ts

```diff
@@ -385,6 +389,40 @@ interface ModuleDefinitionArguments extends Array<any> {
 		pendingCacheInsert = {};
 	}
 
+	function loadNodeModule(moduleId: string, parent?: Module): any {
```

Comment:
We should still find a way to `has()` feature this.  One simple solution is to create a `let loadNodeModule: (moduleId: string, parent?: Module) => any = () => { };` and then assign it where it originally was.  If we have a `noop` handle somewhere, that would be better...